index.js

```diff
@@ -56,6 +58,22 @@ async function checkTargetBranch() {
             console.log(msg);
             await postCommentIfBaseChanged(msg);
         }
+    } else if (isFixBranch(head)) {
+        if (isSemestryTestingBranch(base)) {
+            console.log("Head branch is a fix branch and targets a Semestry version-test branch.");
+        } else {
+            let msg = "⛔ Pull requests for fix branches should target a version-test branch.";
+            console.log(msg);
+            await postCommentIfBaseChanged(msg);
+        }
+    } else if (isStagingBranch(head)) {
+        if (isSemestryStagingBranch(base)) {
+            console.log("Head branch is a staging branch and targets a Semestry version-staging branch.");
+        } else {
+            let msg = "⛔ Pull requests for staging branches should target a staging-test branch.";
+            console.log(msg);
```

Comment:
A `staging-test` branch? This terminology is not on the Wiki.

Reply:
Well observed, I have accidentally named this the wrong way